{"is_answered": true, "view_count": 439, "tags": ["python", "pandas", "google-bigquery", "google-cloud-datalab"], "last_activity_date": 1473514600, "answer_count": 1, "creation_date": 1465829019, "score": 2, "link": "http://stackoverflow.com/questions/37792709/accessing-big-query-from-cloud-datalab-using-pandas", "accepted_answer_id": 37802533, "owner": {"user_id": 137783, "profile_image": "https://www.gravatar.com/avatar/938a1132d5f1b2c5a54345b0e672689c?s=128&d=identicon&r=PG", "user_type": "registered", "reputation": 2766, "link": "http://stackoverflow.com/users/137783/dartdog", "accept_rate": 89, "display_name": "dartdog"}, "title": "Accessing Big Query from Cloud DataLab using Pandas", "last_edit_date": 1466022713, "question_id": 37792709}